### ISSUE TITLE
Can't seem to connect to liteserv

### ISSUE THREAD

**Issue by vasticles** [couchbase/couchbase-lite-android-liteserv#37]
Probably a newb question, but I'm having no luck connecting to liteserv that runs on an emulator.

Vass-MacBook-Pro:~ vas$ curl 192.168.57.101:8081
curl: (7) Failed to connect to 192.168.57.101 port 8081: Operation timed out

8081 is the port I set. I tried the default 5984 as well.
Pings without issues. What am I doing wrong?

**Comment by hideki** [comment]
@vasticles 

Are you using GennyMotion?

First can you check if LiteServ is accessible from browser in emulator?

**Comment by vasticles** [comment]
@hideki 
Yes, I am using GennyMotion.
Emulator browser localhost:8081 seems to work. Shows:
{"couchdb":"Welcome","CBLite":"Welcome","version":"1.2.1"}

**Comment by hideki** [comment]
HI @vasticles,
I am testing product with same environment, LiteServ on GennyMotion, and accessing it from other GennyMotion. It works with my environment.

```
Vass-MacBook-Pro:~ vas$ curl 192.168.57.101:8081
curl: (7) Failed to connect to 192.168.57.101 port 8081: Operation timed out
```
error message says `Failed to connect`. I guess `192.168.57.101:8081` is not accessible from curl command. Please check GennyMotion setting and also VirtualBox setting?

**Comment by vasticles** [comment]
@hideki 
Strange. I tried other ports with same ip, and I get an instant response "Operation refused". The port that is actually supposed to be listening times out like before.
VirtualBox settings are set to Host-only adapter/vboxnet1. Couldn't find anything relevant in GennyMotion settings.

I tried running LiteServ on a real device and had no issues using curl with it. I guess I'll stick to that as it suits my purpose for now.
Thanks.
